addons/point_of_sale/static/src/css/pos.css

```diff
@@ -38,6 +38,13 @@ div.modal.in {
     left: 50%;
 }
 
+@media screen and (max-width: 768px) {
```

Comment:
add in commit message step that lead to the issue (how to reproduce it before changes): The Product Configurtion modal is too wide, and the extra price tag does not look smooth (idea: extra padding between rows or smaller price tag)

Reply:
Yeah my bad